sky/packages/sky/lib/src/animation/performance.dart

```diff
@@ -51,30 +51,30 @@ abstract class WatchableAnimationPerformance {
 /// may also take direct control of the timeline by manipulating [progress], or
 /// [fling] the timeline causing a physics-based simulation to take over the
 /// progression.
-class AnimationPerformance implements WatchableAnimationPerformance {
-  AnimationPerformance({ this.duration, double progress }) {
+class Performance implements PerformanceView {
```

Comment:
Animation is way too vague, IMHO. I'm not a huge fan of Performance either, but I don't think there'd be that much confusion with performance. Maybe I'm not imaginative enough though. :-)